api/middleware.go

```diff
@@ -82,10 +82,13 @@ func MetricsMiddleware(m metrics.RequestMetrics, logger log.Logger) func(next ht
 				// like we're doing here if we don't want to use it.
 				timer.ObserveDuration()
 			}
+			latency := time.Since(t)
 			logger.Info("ending request",
 				"endpoint", r.URL.Path,
+				"query_params", r.URL.RawQuery,
 				"request_id", requestID,
-				"time", time.Since(t),
```

Comment:
:astonished:  It was really unexpected to me that GCP just swallows the "time" field. Nice find. Just leaving a comment for posterity :)